package.json

```diff
@@ -1,5 +1,32 @@
 {
+  "name": "CiviHR",
+  "description": "HR software for non-profit",
+  "license": "AGPL-3.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/civicrm/civihr.git"
+  },
+  "bugs": {
+    "url": "https://github.com/civicrm/civihr/issues"
+  },
+  "scripts": {
+    "precommit": "lint-staged"
+  },
+  "lint-staged": {
+    "*.js": [
+      "semistandard --fix",
+      "git add"
+    ]
+  },
   "semistandard": {
-    "globals": [ "CRM", "inject" ]
+    "globals": [
+      "CRM",
+      "inject"
+    ]
+  },
+  "devDependencies": {
+    "husky": "^0.14.3",
```

Comment:
Should not the versions be fixed?

Reply:
no need for that, since we're using package-lock files